test/DefaultCluster.Tests/ClientAddressableTests.cs

```diff
@@ -7,8 +7,9 @@
 using UnitTests.GrainInterfaces;
 using Xunit;
 
-namespace UnitTests
+namespace DefaultCluster.Tests
 {
+    [TestCategory("BVT")]
```

Comment:
There are a ton of similar instances. I won't mention them all, but please review each of them